S2S Integration Samples/README.md

```diff
@@ -35,4 +35,12 @@ S2SIntegration_ProcessPrintTask is used for
 - Receiving a webhook notification when the Task is created.
 - Getting a Print Job for the task and reading the job attributes.
 - Updating the job's "Color" attribute to "GrayScale".
-- Completing the Task.
\ No newline at end of file
+- Completing the Task.
+
+## Demos
+- Demo: [Create Printer]()
+- Demo: [Hold print job and implement task trigger]()
+
+## Resources
+Code sample to generate a [Certificate Signing Request](https://docs.microsoft.com/en-us/universal-print/hardware/universal-print-oem-certificate-signing-request) with the BouncyCastle C# library.
```

Comment:
Add one more doc resource for application scopes and point to Universal Print Graph API permission scopes

Reply:
Added. Thanks.